blackbox/docs/sql/administration/user_management.txt

```diff
@@ -0,0 +1,109 @@
+.. _administration_user_management:
```

Comment:
if you want to link to the page instead of the headline, you can use the :doc: reference directive